client/components/header/styles.js

```diff
@@ -127,34 +127,23 @@ export const MobileMenuButton = styled.div`
 
 export const MobileMenuWrapper = styled.div`
   position: fixed;
-  top: 0;
+  top: 88px;
   right: 0;
-  background: ${props => props.theme.colors.light};
+  padding: 18px 0;
+  background: ${props => props.theme.colors.grey};
   width: calc(100% - 80px);
   max-width: 350px;
   height: 100%;
   z-index: 600;
```

Comment:
Please check the z-index again, currently clicking on menu items is not working 😄

Reply:
Guess that's an important part of the menu =/ 

I fixed it in a new commit. I also removed the bg color and opacity of the overlay as soren mentioned below. If there is anything else, let me know!